lib/beaker/ssh_connection.rb

```diff
@@ -62,8 +62,14 @@ def connect
     def close
       begin
         @ssh.close if @ssh
-      rescue
-        @ssh.shutdown!
+      rescue => e
+        @logger.warn "Attemped ssh.close.  Caught an error: #{e.message}  Attempting ssh.shutdown!..."
+        begin
+          @ssh.shutdown!
+        rescue => e
+          @logger.warn "Attemped ssh.shutdown!.  Caught an error: #{e.message}. Giving up and destorying ssh."
+          @ssh = nil
+        end
```

Comment:
I would prefer to see catching a specific error instead of the general 'rescue' - we are moving away from empty rescues to explicit rescue conditions.